lib/assessment.js

```diff
@@ -13,14 +20,25 @@ var min = function min() {
 // and returns the sum of that set
 // assume that all the argument are numbers
 var sum = function sum() {
+  var result = 0;
+  for (var i = 0; i < arguments.length; i++) {
+    result += arguments[i];
+  }
+
+  return result;
 
 };
 
 // Question 3
 // Write a function that returns an array of the specified length
 // with all the elements set to defaultValue
-var newArray = function newArray(length, defaultValue) {
+var newArray = function newArray(length, defaultValue){
+  var result = [];
+  for (var i = 0; i < length; i++) {
+    result.push(defaultValue);
+  }
 
+  return array;
```

Comment:
What is `array`? You don't have a variable by that name in this function.

3/4